dali/python/backend_impl.cc

```diff
@@ -33,6 +33,14 @@
 namespace dali {
 namespace python {
 
+// add this alignment to work around a patchelf bug/feature which
+// changes TLS alignment and break DALI interoperability with CUDA RT
+alignas(0x1000) thread_local volatile bool __backend_impl_force_tls_align;
+
+void __backend_impl_force_tls_align_fun(void) {
```

Comment:
DLL_PUBLIC

Reply:
It has own CMake so I would prefer not to use it. I used that function somewhere. Even if it is inlined it should still keep the variable alive.